ui/app/templates/vault/cluster/access/identity/aliases/add.hbs

```diff
@@ -2,7 +2,7 @@
   <p.levelLeft>
     <h1 class="title is-3">
       Create
-      {{lowercase (humanize this.model.identityType)}}
+      {{titleize (humanize this.model.identityType)}}
```

Comment:
In case you want context:
![image](https://github.com/hashicorp/vault/assets/6618863/aa292585-a3dd-4310-89f8-02c578822bbc)